torch/_inductor/ir.py

```diff
@@ -4485,17 +4485,32 @@ def is_not_write(arg):
         self.cpp_kernel_key = f"{self.cpp_kernel_name.replace('::', '_')}_{self.cpp_kernel_overload_name}"  # type: ignore[union-attr]
 
         self.cpp_op_schema = get_cpp_op_schema(kernel)
+        self.init_args_default_value(kernel._schema)
         self.ordered_kwargs_for_cpp_kernel = [
             x.name for x in kernel._schema.arguments if x.kwarg_only
         ]
 
     def is_legacy_abi_kernel(self):
         return "_scaled_dot_product_flash_attention" in str(self.python_kernel_name)
 
+    def init_args_default_value(self, schema):
+        self.args_default_value = [
+            {
+                "name": x.name,
+                "type": x.real_type,
+                "value": x.default_value,
+            }
+            for x in schema.arguments
+            if not x.kwarg_only
+        ]
+
     def get_pos_arg_value(self, pos, kwargs):
         # positional args may be provided in kwargs
         pos_arg_name = self.args_default_value[pos]["name"]
         if pos_arg_name in kwargs:
+            log.debug(
+                f"Found argument {pos_arg_name} with value {kwargs[pos_arg_name]} from kwargs"  # noqa: G004
```

Comment:
nit - I think we discourage using f-strings in log statements.

Reply:
Thanks! I changed this in the next exported diff :)